engine/profiler/src/profile_render.cpp

```diff
@@ -350,10 +355,13 @@ namespace dmProfileRender
         const Area header_area  = GetHeaderArea(display_mode, profiler_area);
         const Area details_area = GetDetailsArea(display_mode, profiler_area, header_area);
 
+        ProfilerFrame empty_frame;
         ProfilerFrame* frame = render_profile->m_ActiveFrame ? render_profile->m_ActiveFrame : render_profile->m_CurrentFrame;
+        frame = frame ? frame : &empty_frame;
```

Comment:
If there is no current frame to render, use an empty one.
(avoids a lot of extra if-statements)